README.asciidoc

```diff
@@ -178,6 +178,14 @@ IMAGE OPTIONS
 
 	 scale image (preserve aspect) until width matches invoked screen
 
+*-gx, --gravity-x*::
+
+	 set the image gravity center in the X axis at position (which can be a fixed amount or a percentage of the image width)
```

Comment:
Can you specify the expected domain/type of the allowed values? e.g. fixed amount should be an integer; percentage a float between 0 and 1 (I prefer this to allowing users to suffix it with a "%", as that simplifies the argument parsing)

Reply:
It still isn't quite clear to me what `position` and `percentage` are relative to - the monitor, or the wallpaper?

e.g. if position is 200, is the image centered at the point 200 pixels right of the image's left edge, or the monitor's left edge?

Similarly, if percentage is 0.2, is the image centered at a point 0.2 * (image width) pixels right of the monitor's edge? Or the wallpaper's edge?

Reply:
`position` and `percentage` are relative to the image: they refer to the image gravity center. Let me explain: when set to zoom, setroot zooms the image until it fits in the screen. The hardcoded behavior right now is to make the image center match the screen center (the image is cut off equally on both sides). This sometimes is not the desired behavior: my use case is an image where the focal point is not in the center, and the result looks very awkward on some screens.

Here I introduce the concept of 'gravity center' of the image: we default to a gravity center at x=50%,y=50% (just like the hardcoded values), but we allow moving that center to another place of the image. I've chosen the term 'gravity center' (shortened to 'gravity') because it's the term I've seen it's used in some other programs to name the same concept. [This Squarespace help page](https://support.squarespace.com/hc/en-us/articles/205826028-Using-focal-points-to-center-images) contains images which are good examples of the benefits of changing the image center.

Looking at the surrounding documentation, it seems like too much information for a flag description. Moreover it would be duplicated (for the X and Y axis). What do you think about grouping gravity-related information in a separate section? I think it would make sense.

As for the percentage, it's so it can be distinguished from absolute values (should `--gravity-x 1` mean the gravity center is at x=1? or at x=100%?) Maybe it's not so relevant (we could consider values >1 as absolutes and values <1 as percentages, but that gives us that single problematic value, and the resulting interface would be a bit awkward).

Reply:
I agree with what you have said. I'll merge after you make the commit. Thanks for your contribution!